packages/theia-integration/package.json

```diff
@@ -44,7 +44,11 @@
     "watch": "tsc -w"
   },
   "dependencies": {
-    "@eclipse-glsp/client": "next"
+    "@eclipse-glsp/client": "next",
+    "ws": "^8.13.0"
```

Comment:
We should probably align this with the versions used in Theia.
I quickly checked and Theia uses:

`"ws:":"~8.11.0"`